src/ec/packages/ec-component-text-area/text-area.html.twig

```diff
@@ -0,0 +1,83 @@
+{#
+  Parameters:
+  - "id" (string) (default: '')
+  - "disabled" (boolean) (default: false)
+  - "invalid" (boolean) (default: false)
+  - "name" (string) (default: '')
+  - "hide_label" (boolean) (default: '')
+  - "placeholder" (string) (default: '')
+  - "rows" (int) (default: 4)
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the text area
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for text area
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+  Blocks:
+  - "helper_text"
+  - "invalid_text"
+  - "label"
+#}
+
+{# Internal properties #}
+{% set _id = id|default('') %}
+{% set _disabled = disabled|default(false) %}
+{% set _invalid = invalid|default(false) %}
+{% set _name = name|default('') %}
+{% set _hide_label = hide_label|default(false) %}
+{% set _placeholder = placeholder|default('') %}
+{% set _rows = rows|default(4) %}
+
+{% set _helper_text = helper_text|default('') %}
+{% set _invalid_text = invalid_text|default('') %}
+{% set _label = label|default('') %}
+
+{% set _css_class = ['ecl-text-area'] %}
+{% if _invalid %}
+  {% set _css_class=_css_class|merge(['ecl-text-area--invalid']) %}
+{% endif %}
+{% set _css_class=_css_class|join(' ') %}
+
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+{% spaceless %}
+<div class="ecl-form-group ecl-form-group--text-area">
+  {%- if _label is not empty -%}
+    <label 
+      class="ecl-form-label{{- invalid ? ' ecl-form-label--invalid' -}}{{- hide_label ? ' ecl-form-label--hidden' -}}"
+      for={{- _id -}}
+    >
+      {%- block label _label -%}
+    </label>
+  {%- endif -%}
+
+  <textarea
+    id="{{- _id -}}"
+    name="{{- _name -}}"
+    rows="{{- _rows -}}"
+    placeholder="{{- _placeholder -}}"
+    class="{{- _css_class -}}"
+    {{- _disabled ? "disabled" -}}
+    {{- _extra_attributes|raw -}} 
```

Comment:
Quickly checked a very-old implementation. It has `aria-multiline` and `role="textbox"` attributes. Seems like they could be useful for [assisting technologies](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role), would it make sense to include them by default?

Reply:
Good point... I don't know if they have been remove on purpose in ECL 2.
I'll check

Reply:
As it seems, the `role="textbox"` should not be used in textarea (first paragraph [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role), and also https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#Technical_summary)

In a similar way, `aria-multiline` seems to be used only to mimic a textarea, but not in actual textareas.
So I'd leave the implementation as it is.